<launch>
  <node pkg="ais_robot_localization" type="navsat_preprocessing_node.py" name="navsat_preprocessing_node" output="screen">
    <!-- Load parameters from YAML -->
    <rosparam file="$(find ais_robot_localization)/params/navsat_preprocessing_geo_kombi.yaml" command="load" />

    <!-- Topic remapping -->
    <remap from="/localization/preprocessing/input/gps_odometry" to="/odom_gnss" />
    <remap from="/localization/preprocessing/input/orientation_with_global_heading" to="/odom_gnss_orientation" />
    <remap from="/localization/preprocessing/input/local_reference" to="/odom" />
    <remap from="/localization/preprocessing/output/odometry" to="/localization/preprocessing/global_odom_fix_Cov" />
    <remap from="/localization/preprocessing/output/imu_with_fix_cov" to="/localization/preprocessing/global_orientation_fix_Cov" />

    <!--<remap from="/localization/preprocessing/state" to="/state_remapped" />
    <remap from="/localization/preprocessing/input/orientation" to="" />  -->
  </node>
</launch>
